index.html

```diff
@@ -5332,7 +5332,7 @@ <h2><span class="chunk">fcTL</span>
 
 </section>
 
-<section>
+<section id="11fdAT">
```

Comment:
Same here

Reply:
This is to preserve anchors from previous editions of the specification.